### ISSUE TITLE
Internal server error on homepage

### ISSUE THREAD

**Issue by hackdna** [parklab/refinery-platform#863]
Running commit 3237c59 on refinery-dev. Homepage displays Internal Server Error. Apache error log:
```
fatal: Not a git repository (or any of the parent directories): .git
Unknown option: -C
usage: git [--version] [--exec-path[=GIT_EXEC_PATH]] [--html-path]
           [-p|--paginate|--no-pager] [--no-replace-objects]
           [--bare] [--git-dir=GIT_DIR] [--work-tree=GIT_WORK_TREE]
           [--help] COMMAND [ARGS]
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208] mod_wsgi (pid=15885): Exception occurred processing WSGI script '/srv/scc/apps/refinery-platform/refinery/config/wsgi.py'.
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208] Traceback (most recent call last):
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]   File "/srv/scc/apps/refinery-platform/refinery/config/wsgi.py", line 15, in application
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]     return _application(environ, start_response)
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]   File "/srv/scc/virtualenvs/refinery-platform/lib/python2.7/site-packages/django/core/handlers/wsgi.py", line 236, in __call__
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]     self.load_middleware()
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]   File "/srv/scc/virtualenvs/refinery-platform/lib/python2.7/site-packages/django/core/handlers/base.py", line 45, in load_middleware
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]     for middleware_path in settings.MIDDLEWARE_CLASSES:
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]   File "/srv/scc/virtualenvs/refinery-platform/lib/python2.7/site-packages/django/conf/__init__.py", line 53, in __getattr__
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]     self._setup(name)
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]   File "/srv/scc/virtualenvs/refinery-platform/lib/python2.7/site-packages/django/conf/__init__.py", line 48, in _setup
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]     self._wrapped = Settings(settings_module)
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]   File "/srv/scc/virtualenvs/refinery-platform/lib/python2.7/site-packages/django/conf/__init__.py", line 132, in __init__
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]     mod = importlib.import_module(self.SETTINGS_MODULE)
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]   File "/srv/scc/virtualenvs/refinery-platform/lib/python2.7/site-packages/django/utils/importlib.py", line 35, in import_module
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]     __import__(name)
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]   File "/srv/scc/apps/refinery-platform/refinery/config/settings/prod.py", line 3, in <module>
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]     from .base import *
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]   File "/srv/scc/apps/refinery-platform/refinery/config/settings/base.py", line 471, in <module>
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]     "--pretty=format:%H", '-n', '1'])
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]   File "/n/sw/centos6/python-2.7.3/lib/python2.7/subprocess.py", line 544, in check_output
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208]     raise CalledProcessError(retcode, cmd, output=output)
[Tue Feb 02 16:26:54 2016] [error] [client 134.174.140.208] CalledProcessError: Command '['/usr/bin/git', '-C', '/vagrant/refinery', 'log', '--pretty=format:%H', '-n', '1']' returned non-zero exit status 129
```

**Comment by hackdna** [comment]
Option `-C` is not supported by git installed on refinery-dev (version 1.7.1)

**Comment by ngehlenborg** [comment]
Probably time for an update: https://github.com/git/git/releases

**Comment by hackdna** [comment]
Fixed in 132ca9d2
